core/domain/exp_services.py

```diff
@@ -941,14 +1190,32 @@ def save_exploration_summary(exp_summary):
 
 
 def delete_exploration_summary(exploration_id):
-    """Delete an exploration summary model."""
+    """Delete an exploration summary model.
+
+    Args:
+        exploration_id: str.The exploration id of the exploration 
```

Comment:
This docstring is misleading. It's a storage model and not a domain object that's being deleted. It's also an ExplorationSummary and not an Exploration that's being deleted.

exploration_id: str. The id of the exploration summary to be deleted.